tap_s3_csv/discover.py

```diff
@@ -1,24 +1,54 @@
+"""
+Discovery mode is connecting to the data source and collecting information that is required for running the tap.
+"""
+from typing import List, Dict
+
 from singer import metadata
 from tap_s3_csv import s3
 
-def discover_streams(config):
+
+def discover_streams(config: Dict)-> List[Dict]:
+    """
+    Run discovery mode for every stream in the tap configuration
+    :param config: connection and streams configuration
+    :return: list of information  about every stream
+    """
     streams = []
 
     for table_spec in config['tables']:
         schema = discover_schema(config, table_spec)
-        streams.append({'stream': table_spec['table_name'], 'tap_stream_id': table_spec['table_name'], 'schema': schema, 'metadata': load_metadata(table_spec, schema)})
+        streams.append({'stream': table_spec['table_name'],
+                        'tap_stream_id': table_spec['table_name'],
```

Comment:
'stream' and 'tap_stream_id' are the same?

Reply:
tap-stream-In the case of tap s3 csv, yes, there are the same, because we don't have a database nor schema. 
In tap-postgres for example: stream pattern would be `{{table-name}}` and tap-stream-id pattern would `{{schema_name}}-{{table_name}}`.